cpp/kiss_icp/core/Deskew.cpp

```diff
@@ -37,8 +37,9 @@ namespace kiss_icp {
 std::vector<Eigen::Vector3d> DeSkewScan(const std::vector<Eigen::Vector3d> &frame,
                                         const std::vector<double> &timestamps,
                                         const Sophus::SE3d &start_pose,
-                                        const Sophus::SE3d &finish_pose) {
-    const auto delta_pose = (start_pose.inverse() * finish_pose).log();
+                                        const Sophus::SE3d &finish_pose,
+					int prev_frame_delta) {
+    const auto delta_pose = (start_pose.inverse() * finish_pose).log() / prev_frame_delta;
```

Comment:
We may want to check if `prev_frame_delta` equals to zero

Reply:
addressed